tools/clang/test/CodeGenDXIL/hlsl/intrinsics/linalg_builtins/linalg-builtins.hlsl

```diff
@@ -0,0 +1,59 @@
+// RUN: %dxc -fcgl -T cs_6_9 -E cs_main %s | FileCheck %s
+
+ByteAddressBuffer matrix_buffer;
+ByteAddressBuffer bias_buffer;
+RWByteAddressBuffer rw_matrix_buffer;
+
+[Shader("compute")]
+[NumThreads(1,1,1)]
+void cs_main()
+{	
+	vector<float, 4> output_vector;
+	static const uint is_output_unsigned = 0;
+	
+	vector<float, 4> input_vector;
+	const uint is_input_unsigned = 0;
+	const uint input_interpretation = 9; /*F32*/
+	
+	const uint matrix_offset = 0;
+	const uint matrix_interpretation = 9; /*F32*/
+	const uint matrix_dimM = 4;
+	const uint matrix_dimK = 4;
+	const uint matrix_layout = 0; /*RowMajor*/
+	const bool matrix_is_transposed = false; 
+	const uint matrix_stride = 64;
+
+	// CHECK: call void @"dx.hl.op..void (i32, <4 x float>*, i1, <4 x float>, i1, i32, %dx.types.Handle, i32, i32, i32, i32, i32, i1, i32)"(i32 363,
```

Comment:
Shouldn't these CHECKs check the arguments?

Reply:
This has been addressed, see https://github.com/microsoft/DirectXShaderCompiler/pull/7290/files#diff-ade942e9326b532aaa48b48b208b236d6de1227c542da1010b67f46a957d0df4
(Clicking on the file name in the thread title got me to a stale version of that file)